code/modules/mob/living/simple_animal/hostile/retaliate/exoplanet.dm

```diff
@@ -1,45 +1,54 @@
 
 /mob/living/simple_animal/hostile/retaliate/beast
-	var/hunger = 0
-	var/list/prey = list()
+	ai = /datum/ai/beast
+	var/nutrition = 300
```

Comment:
See linter check; this is a redeclaration of the existing nutrition var. Remove the declaration or just make it `nutrition = 300`?